github/projects.go

```diff
@@ -285,15 +285,26 @@ type ProjectCard struct {
 	CreatedAt  *Timestamp `json:"created_at,omitempty"`
 	UpdatedAt  *Timestamp `json:"updated_at,omitempty"`
 	NodeID     *string    `json:"node_id,omitempty"`
+	Archived   *bool      `json:"archived,omitempty"`
 
 	// The following fields are only populated by Webhook events.
 	ColumnID *int64 `json:"column_id,omitempty"`
 }
 
+// ProjectCardListOptions specifies the optional parameters to the
+// ProjectsService.ListProjectCards method.
+type ProjectCardListOptions struct {
+	// ArchivedState is used to list all,archived, or not_archived project cards.
+	// Defaults to not_archived when you omit this parameter.
+	ArchivedState string `url:"archived_state,omitempty"`
```

Comment:
s/string/*string/
so that the option can be omitted.

Reply:
Shall I make this change all in existing options struct?

Reply:
No, let's just address this one in this PR.
Since this is an optional parameter, we need to make sure that it can be unset, and a pointer is the way to do that.